linux_os/guide/system/accounts/accounts-session/accounts_tmout/ansible/sle12.yml

```diff
@@ -0,0 +1,15 @@
+# platform = multi_platform_sle
```

Comment:
I think these `accounts_tmout` can be removed as per: https://github.com/ComplianceAsCode/content/pull/6631#discussion_r584669502

Reply:
@ggbecker I don't think so in this case. As I read shared remediation, the variables will be written to /etc/profile and that would be different from our normal set up. The oval in 6631 just meant that we would pass if the var was in /etc/profile or in some. sh under /etc/profile.d which is fine. But I don't think we want remediation to put it in /etc/profile by default.

Reply:
All right. It's ok to leave these changes then. Thanks for the explanation.